### PR TITLE
Readme touchup

### PR DESCRIPTION
**Pull request by edgargdev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,10 +3,32 @@
 This project is generated with [yo angular generator](https://github.com/yeoman/generator-angular)
 version 0.15.1.
 
+## Getting Started
+1. git clone
```
Comment (michaelachrisco) [inline README.md:7]: This should probably be `git clone git@github.com:freecodecamp-fresnoca/arthop-map-dev.git` for ease of use.

File: README.md [hunk 2]
```diff
@@ -3,10 +3,32 @@
...
+  * If node < 6.4.0 install using nvm
+  * `nvm use 6.4.0` or `nvm install 6.4.0` 
+  * if nvm is not installed on your machine
+    * Installation instructions [bash] (https://github.com/creationix/nvm) [zsh] (https://github.com/lukechilds/zsh-nvm#manually)
+4. Make sure nodemon is installed `nodemon --version`
+  * To install run `npm install -g nodemon`
+5. Run `npm install` to install dependencies
+6. Run `cd app/public/ && bower install`
+  * Remember to `cd ../..` to get back to root directory
+  * If bower is not globally installed run `npm install -g bower`
```
Comment (michaelachrisco) [inline README.md:19]: My suggestion would be to Add a Dependencies section to the docs. Something like:

## Dependencies
- Node 6.4 or higher
- MongoDB install
- Global install of Bower and nodemon `npm install -g nodemon bower`

File: README.md [hunk 3]
```diff
@@ -3,10 +3,32 @@
...
+    * [Linux] (https://docs.mongodb.com/v3.0/administration/install-on-linux/)
+9. Run `npm start` to start node server. 
+10. Application will be found at localhost:3333
+
 ## Build & development
 
-Run `npm install` and `bower install`<br/>
-Run `grunt` for building and `grunt serve` for preview.
+Run `npm install` and `cd app/public/ && bower install`<br/>
+Run `npm start` to get node server running
```
Comment (michaelachrisco) [inline README.md:31]: mongo + npm start would be good to script out as a start script. Might be a good github ticket :smile:

File: bower.json [hunk 4]
```diff
@@ -1,27 +0,0 @@
-{
```
Comment (michaelachrisco) [inline bower.json:1]: So npm is taking care of this bower package json now correct?

**Review by michaelachrisco — COMMENTED** [review]
Always love more documentation! Great job. Just a couple of suggestions for you @Rawradin. :+1:

**Comment by rxsharp** [comment]
@michaelachrisco, this shouldn't have any issues with all the new changes you are making in the testing branch?

**Comment by michaelachrisco** [comment]
Nope. Its all good!

**Pull request merged**